distributions/aws/examples/rds-s3/README.md

```diff
@@ -102,33 +118,6 @@ kubectl apply -f https://raw.githubusercontent.com/aws/secrets-store-csi-driver-
           minioServiceHost=s3.amazonaws.com
           minioServiceRegion=us-west-2
           ```
-    1. Configure an AWS secret with your RDS database username and password that were configured when following the steps in Create RDS Instance.
-        - For example, if your username is `admin` and your password is `Kubefl0w` then your secret should look like:
-        - **Note:** These are the default values for database credentials in cloudformation template for creating the RDS instance, change these according to the values you used
-        - ```
-          aws secretsmanager create-secret --name rds-secret --secret-string '{"username":"admin","password":"Kubefl0w"}'  --region $CLUSTER_REGION
-          ```
-    1. Configure an AWS secret with your AWS credentials. These need to be long term credentials from an IAM user and not temporary. 
-        - Find more details about configuring/getting your AWS credentials here:
-        https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html
-        - ```
-          aws secretsmanager create-secret --name s3-secret --secret-string '{"accesskey":"AXXXXXXXXXXXXXXXXXX6","secretkey":"eXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXq"}'  --region $CLUSTER_REGION
-          ```
-
-### 3. Configure Katib
```

Comment:
you will need to update the cognito-rds-s3 readme according to the changes in this readme like configure Katib section does not exist anymore

mainly the first 3 sections, Prerequisites, Configure Kubeflow Pipelines for RDS and S3 and Configure Katib for RDS